chromium_src/components/content_settings/core/browser/cookie_settings.h

```diff
@@ -10,6 +10,7 @@
 #include <vector>
 
 #include "base/containers/flat_map.h"
+#include "brave/components/brave_shields/core/common/shields_settings.mojom.h"
```

Comment:
this breaks the build because there is no dependency on the mojom file in the `cookie_settings` target